### PR TITLE
Update mobile-installation.md

### PR DESCRIPTION
**Pull request by david-kariuki**
I have proposed some correction to the content page of the developer-portal mobile-installation section. I have proposed the edits to make the instructions simpler for everyone and especially newbies.

### CODE REVIEW

File: start/sw/mobile-app/mobile-installation.md [hunk 1]
```diff
@@ -6,26 +6,28 @@ order: 1
 description: Creating mobile applications using Google Android Studio.
 ---
 
-# Mobile application
+# Mobile application.
```
Comment (pvalena) [inline start/sw/mobile-app/mobile-installation.md:9]: We're not using dots for headers.

**Review by pvalena — CHANGES_REQUESTED** [review]
Thanks for the rewording, but please drop the changes adding `.` for headers.

**Comment by jackorp** [comment]
Ping... @david-kariuki Are you still interested in advancing this PR?

**Review by naizar86 — APPROVED** [review]
> # @`
Duplicate of #`

**Comment by jackorp** [comment]
If anyone is interested in advancing this PR, feel free to resubmit.

**Comment by david-kariuki** [comment]
No

**Pull request closed without merging**